source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,865 @@
+all files in an update, as well as zero- and delta-packs for improved update
+performance. The metadata, stored as manifests, describes all of the bundle
+information for the update. Update content produced by mixer is then published
+to a web server and consumed by clients via swupd. Refer to
+:ref:`swupd <swupd-guide>` for additional information regarding updates and
+update content.
 
-There are two different workflows to create your own mix.
-First, if your mix only uses |CL| content, *skip Create custom RPMs* below. 
-Second, if your mix includes your own 
-:abbr:`RPMs (RPM Package Manager files)`, follow all steps below. 
+How it works
+************
 
-.. contents:: 
+Learn the mixer tool set up and workflow.
+
+.. contents::
    :local:
    :depth: 1
-   :backlinks: top
-
-The following sections contain detailed information on every step of
-these workflows.
 
-.. _create-nginx-web-server:
-
-Create nginx web server to host mixer updates
-=============================================
-
-Follow these steps to set up a HTTP service with ``nginx`` web 
-server, where you can host custom |CL| mixes:  
+Prerequisites
+=============
 
-#. Install ``web-server-basic``. 
-   
-   .. code-block:: bash
+* :command:`mixer` bundle
 
-      sudo swupd bundle-add web-server-basic
+  Add the mixer tool with the :command:`mixer` bundle. Refer to
+  `Install a bundle`_ for more details.
 
-#. Make the directory where mixer updates will reside.  
+* Docker container
 
-   .. code-block:: bash
+  mixer by default runs all build commands in a Docker container to make sure
+  the correct tool versions are used. This also allows custom mixes to
+  automatically perform downstream format bumps when the upstream releases a
+  format bump. See `Format version`_ for additional information regarding
+  format bumps.
 
-      sudo mkdir -p /var/www
+  Refer to `Configure and enable Docker`_ for instruction.
 
-#. Create a symbolic link.
+* Docker proxy (optional)
 
-   .. code-block:: bash
+  If you use a proxy server, you must set your proxy environment variables and
+  create a proxy configuration file for the Docker daemon and container.
 
-      sudo ln -sf /home/clr/mix/update/www /var/www/mixer
+  Consult your IT department if you are behind a corporate proxy for the correct
+  values.
 
-   .. note::
+  Refer to `Configure Docker proxy info`_ for instruction.
 
-      `/home/clr/mix` is the workspace for the mix (read `Create workspace`)
+* Location to host the update content and images
 
+  In order for swupd to make use of your mix, the update content for your mix
+  must be hosted on a web server. Your mix will be configured with an update
+  location URL, which swupd will use to pull down updates.
 
-#. Set up ``nginx`` configuration. 
+  Refer to `Set up a nginx web server for mixer`_ for an simple example of
+  setting up an update location.
 
-   .. code-block:: bash
+First time setup
```

Comment:
"First-time setup" doesn't seem like the right header because you create a workspace and initialize the mix.  To me "first-time" suggests to me that you only do this once, which would be installing the mixer bundle.  Perhaps, this should be titled "Initialize a mix".